test/resources/payload.json

```diff
@@ -484,134 +484,52 @@
   ],
   "trustees": [
     {
-      "partnershipDetails": {
-        "name": "Test partnership name",
-        "isDeleted": false
+      "isTrusteeNew" : true,
```

Comment:
Removing existing data might break some tests in 3080. Leave in the partnership please

Reply:
done